core/src/main/java/org/eclipse/hono/config/AbstractConfig.java

```diff
@@ -290,6 +295,42 @@ public final void setKeyPath(final String keyPath) {
         this.keyPath = keyPath;
     }
 
+    /**
+     * Specify the format of the trust store explicitly.
+     * 
+     * @param trustStoreFormat The format to use when reading the trust store, may be {@code null} to trigger auto detection.
+     */
+    public void setTrustStoreFormat(FileFormat trustStoreFormat) {
```

Comment:
do we actually have a problem with the trust store format not being recognized correctly?